css/styles.css

```diff
@@ -58,15 +60,18 @@
     margin: 0;
     font-size: 14px;
     font-weight: normal;
+    line-height: 20px;
     text-transform: uppercase;
 }
 .ab-column h3.icon:before {
-    font: normal 28px/28px icons;
+    font: normal 30px/30px icons;
     top: 4px;
-    right: 0;
+    right: 20px;
     speak: none;
+    width: 1em;
     color: #8e959c;
     position: absolute;
+    text-align: center;
```

Comment:
This, as part of the CSS until 2d193ccc36be60025bd5d8d82f6238b0203b5ffb, affects the icons alignment. It should not have been removed without further investigation.